curriculum/challenges/english/25-front-end-development/lecture-understanding-performance-in-web-applications/67d2f80313efe77e43182e86.md

```diff
@@ -2,13 +2,49 @@
 id: 67d2f80313efe77e43182e86
 title: What Are Some Techniques for Improving JavaScript Performance?
 challengeType: 11
-videoId: nVAaxZ34khk
+videoId: dQHwpQos4vk
 dashedName: what-are-some-techniques-for-improving-javascript-performance
 ---
 
 # --description--
 
-Watch the lecture video and answer the questions below.
+Watch the video or read the transcript and answer the questions below.
+
+# --transcript--
+
```

Comment:
Okay, found another one that doesn't start with the title.

Reply:
Same